core/src/main/java/org/apache/accumulo/core/conf/AccumuloConfiguration.java

```diff
@@ -160,15 +160,17 @@ public long getUpdateCount() {
 
     PrefixProps prefixProps = cachedPrefixProps.get(property);
 
-    if (prefixProps == null || prefixProps.updateCount != getUpdateCount()) {
+    long currentCount = getUpdateCount();
+
+    if (prefixProps == null || prefixProps.updateCount != currentCount) {
       prefixCacheUpdateLock.lock();
       try {
         // Very important that update count is read before getting properties. Also only read it
         // once.
-        long updateCount = getUpdateCount();
+        long startCount = getUpdateCount();
```

Comment:
The name change was held over from a previous approach where a start / end check was used. will change it back.

I think it would more difficult to isolate changes that occurred because of IDE recommendations than reviewing them in this context.  The changes should be easy to determine that they remain correct (or not) and it seems that it would be just as fast / easy to review them once rather than 2 PRs that are changing essentially the same files.  But that's my opinion.

Reply:
Addressed in 64191f7b1d